[TOOLS-4780] Modify the directory of record files extracted by SQL transfer

Kang-dot opened the pull request:
<http://jira.cubrid.org/browse/TOOLS-4780>

### Purpose
- SQL을 통해 추출한 record 파일의 저장 위치 수정

### Implementation
- SQL 이관 record, table 파일이 제대로 생성되지 않는 문제 수정
- SQL 이관 record 파일의 생성 위치를 수정

### Remarks
N/A

Kang-dot committed:
[TOOLS-4704] Changing the Output Method of SQL Migration Tables

- fix SQL migration dump file isn't create issue
- change SQL migration dump file's output location

Kang-dot committed:
[TOOLS-4704] apply formatter

- apply formatter

Kang-dot committed:
[TOOLS-4704] refactoring code

- change code structure

Kang-dot committed:
[TOOLS-4780] fix script migration

Kang-dot committed:
[TOOLS-4780] add description

Srltas reviewed (commented):
1. `__SQLTABLE__/objects/`로 한 단계 더 디렉터리를 나눈 이유가 있을까요? 별도 의미가 없다면 `__SQLTABLE__/` 바로 하위에 출력해도 충분해 보입니다.

2. 테이블별로 레코드 파일을 분리하지 않는 경우에는, 기존 objects 파일(예: demodb_public_objects)에 그대로 기록할 수 없는 건가요? 
DDL은 기존 파일에 생성되는 반면, 레코드는 `__SQLTABLE__/demodb__SQLTABLE__objects` 같은 별도 파일로 분리되어 결과적으로 DDL은 단일 파일에, 레코드는 두 개의 objects 파일에 나뉘어 기록됩니다. 이렇게 동작이 이원화되면서 전체 구조가 불필요하게 복잡해지는 느낌입니다.

Srltas commented:
디렉터리/파일명을 옵션 별로 어떻게 다른지 jira 이슈 또는 PR 본문에 조금만 더 상세히 작성해 주시면 리뷰에 도움이 될 것 같습니다!..

Kang-dot committed:
[TOOLS-4780] restore directory structure

- remove "__SQLTABLE__" directory relate settings

Kang-dot committed:
Merge branch 'CUBRID:develop' into TOOLS-4704-2

Kang-dot committed:
Merge branch 'TOOLS-4704-2' of https://github.com/Kang-dot/cubrid-migration into TOOLS-4704-2

Kang-dot commented:
팀 내부에서 논의 후 파일 출력 구조를 변경합니다.

- split schema
활성화 여부에 관계 없이 기존과 동일하게 object 파일에 일괄 출력

- one table one file
SQL에 의해 추출된 파일 이름 중간에 "_SQLTABLE_"이 들어가도록 수정
ex) dba___SQLTABLE___SQL1

Kang-dot committed:
Merge branch 'CUBRID:develop' into TOOLS-4704-2

Kang-dot committed:
Merge branch 'TOOLS-4704-2' of https://github.com/Kang-dot/cubrid-migration into TOOLS-4704-2

Kang-dot committed:
[TOOLS-4780] refactoring code

- refactoring code
- merge with TOOLS-4776 changes (script structure refactoring issue)
- update code formatter (TOOLS-4785)

Kang-dot committed:
[TOOLS-4780] reapply formatter

Kang-dot committed:
[TOOLS-4780] change "__SQLTABLE__" description
